backend/lambda/lambda.go

```diff
@@ -60,3 +63,30 @@ func (s *Client) GetSessionScreenshot(ctx context.Context, projectID int, sessio
 	_ = signer.SignHTTP(ctx, *s.Credentials, req, NilPayloadHash, string(ExecuteAPI), s.Config.Region, time.Now())
 	return s.HTTPClient.Do(req)
 }
+
+func (s *Client) GetSessionInsight(ctx context.Context, projectID int, sessionID int) (*http.Response, error) {
+	b, _ := json.Marshal(&modelInputs.SessionQuery{
+		ID:        sessionID,
+		ProjectID: projectID,
+	})
+
+	var req *http.Request
+
+	if util.IsDevEnv() {
+		b, _ = json.Marshal(&modelInputs.SessionQuery{
+			ID:        232563428,
+			ProjectID: 1,
+		})
+		req, _ = http.NewRequest(http.MethodPost, "https://ohw2ocqp0d.execute-api.us-east-2.amazonaws.com/default/ai-insights", bytes.NewBuffer(b))
+	} else {
+		req, _ = http.NewRequest(http.MethodPost, "https://ohw2ocqp0d.execute-api.us-east-2.amazonaws.com/default/ai-insights", bytes.NewBuffer(b))
+	}
```

Comment:
pull the `http.NewRequest` out of the `if/else` since it is the same. just the `modelInputs.SessionQuery` needs to change in dev mode.

Reply:
Oh, good catch, it's meant to have the localhost url as the default in dev, but if that request fails it calls the lambda